requirements/test.txt

```diff
@@ -26,7 +26,7 @@ binaryornot==0.4.4
     # via
     #   -r requirements/base.txt
     #   cookiecutter
-bleach==4.1.0
+bleach==5.0.0
```

Comment:
`bleach==5.0.0` contains breaking changes related to `clean()` function used to sanitize css .  Since tests are passing so it can be upgraded.